examples/build_qwen3_decode_expand_mixed_kernel/passes_dump/11_after_InferTileMemorySpace.py

```
@@ -0,0 +1,332 @@
+        sq_sum_3: pl.Tensor[[16, 1], pl.FP32] = ret[3]
+        x_chunk_0: pl.Tensor[[16, 256], pl.FP32] = ret[4]
+        for b0_0, (k0_iter_1, k_proj_iter_1, kb_iter_1, q_proj_iter_1, v_proj_iter_1, x_chunk_iter_1) in pl.range(0, 16, 4, init_values=(k0_0, k_proj_0, kb_0, q_proj_0, v_proj_0, x_chunk_0)):
+            inv_rms_tile_0: pl.Tensor[[4, 1], pl.FP32] = pl.tensor.slice(inv_rms_0, [4, 1], [b0_0, 0])
+            for ob_0_out, (k0_iter_3_outer_l0, kb_iter_3_outer_l0, q_proj_iter_3_outer_l0, x_chunk_iter_3_outer_l0) in pl.parallel(20, init_values=(k0_iter_1, kb_iter_1, q_proj_iter_1, x_chunk_iter_1)):
+                ret_1: pl.Tuple([pl.Scalar[pl.INDEX], pl.Scalar[pl.INDEX], pl.Tensor[[16, 5120], pl.BF16], pl.Tensor[[16, 256], pl.FP32]]) = self.qwen3_decode_layer_incore_1(b0_0, hidden_states_0, input_rms_weight_0, inv_rms_tile_0, k0_iter_3_outer_l0, kb_iter_3_outer_l0, ob_0_out, q_proj_iter_3_outer_l0, wq_0, x_chunk_iter_3_outer_l0)
+                k0_iter_3_outer_l1_rv: pl.Scalar[pl.INDEX] = ret_1[0]
+                kb_iter_3_outer_l1_rv: pl.Scalar[pl.INDEX] = ret_1[1]
+                q_proj_iter_3_outer_l1_rv: pl.Tensor[[16, 5120], pl.BF16] = ret_1[2]
+                x_chunk_iter_3_outer_l1_rv: pl.Tensor[[16, 256], pl.FP32] = ret_1[3]
+                k0_iter_3_outer_l0_rv, kb_iter_3_outer_l0_rv, q_proj_iter_3_outer_l0_rv, x_chunk_iter_3_outer_l0_rv = pl.yield_(k0_iter_3_outer_l1_rv, kb_iter_3_outer_l1_rv, q_proj_iter_3_outer_l1_rv, x_chunk_iter_3_outer_l1_rv)
+            for ob_1_out, (gamma_iter_1_outer_l0, k0_iter_8_outer_l0, k_proj_iter_3_outer_l0, kb_iter_6_outer_l0, normed_iter_1_outer_l0, v_proj_iter_3_outer_l0, x_chunk_iter_8_outer_l0, x_chunk_bf16_iter_1_outer_l0) in pl.parallel(4, init_values=(gamma_0, k0_iter_3_outer_l0_rv, k_proj_iter_1, kb_iter_3_outer_l0_rv, normed_0, v_proj_iter_1, x_chunk_iter_3_outer_l0_rv, x_chunk_bf16_0)):
```

Ruff (0.15.6)

[error] Undefined name `gamma_0`

(F821)